### DISCUSSION TITLE
Safari 18 on Ventura

### DISCUSSION THREAD

**Discussion by kevinpvo** [osy86/HaC-Mini#1112] [General]
I was wondering if anyone had upgraded to Safari 18 on Ventura yet? 

I have been using Safari 17.5 on 13.3.1 with no observable issues thus far. I'm debating taking the jump, and updating to Safari 18, but wanted to see if anyone else had done so already and any advice. Thanks!

**Comment by 1git** [comment]
![Screenshot 2024-09-24 at 3 29 12 PM](https://github.com/user-attachments/assets/d6ed97f0-b418-4854-8fde-b88725a70d35)

I did it day 1, I didn't even notice that I have upgraded....

**Reply by kevinpvo** [reply]
that's great to see - thanks!
